### PR TITLE
Add unified customization for camera flip button

### PR DESCRIPTION
**Pull request by igorkravchenko**
MOB-3292

**Jira issue:**
https://glia.atlassian.net/browse/MOB-3292

**What was solved?**
Add unified customization for camera flip button along with updated accessibility properties.

**Release notes:**

 - [ ] Feature
 - [x] Ignore
 - [ ] Release notes (Is it clear from the description here?)
 - [ ] Migration guide (If changes are needed for integrator already using the SDK - what needs to be communicated? Add underneath please)

**Additional info:**

- [ ] Is the feature sufficiently tested? All tests fixed? Necessary unit, acceptance, snapshots added? Check that at least new public classes & methods are covered with unit tests
 - [ ] Did you add logging beneficial for troubleshooting of customer issues?
 - [ ] **Did you add new logging?** We would like the logging between platforms to be similar. Refer to [**Logging from iOS SDKs** → **Things to consider for newly added logs**](https://glia.atlassian.net/wiki/spaces/ENG/pages/3589734507/Logging+from+iOS+SDKs#Things-to-consider-for-newly-added-logs) in Confluence for more information.

**Screenshots:**

### CODE REVIEW

**Review by gersonnoboa — APPROVED** [review]

File: GliaWidgets/Sources/RemoteConfiguration/RemoteConfiguration.swift [hunk 1]
```diff
@@ -36,14 +36,20 @@ extension RemoteConfiguration {
...
         enum CodingKeys: String, CodingKey {
             case background, bottomText, buttonBar, duration, header, connect
             case callOperator = "operator"
             case topText
+            case visitorVideo
         }
     }
 
+    struct VisitorVideo: Codable {
+        let flipCameraButton: BarButtonStates?
```
Comment (EgorovEI) [inline GliaWidgets/Sources/RemoteConfiguration/RemoteConfiguration.swift:50]: @AndriiHorishniiMOC has opened [PR](https://github.com/salemove/mobile-sdk-unified/pull/36/files) to remove redundant button styles to have only one state. 
@igorkravchenko are you going to align it on WidgetsSDK side in next PR?

Comment (igorkravchenko) [reply GliaWidgets/Sources/RemoteConfiguration/RemoteConfiguration.swift:50]: Sure, we had discussion with Andrii about it. Thank you for your help with this PR 🤝

**Review by rasmustautsglia — APPROVED** [review]

File: GliaWidgets/Sources/RemoteConfiguration/RemoteConfiguration.swift (hunk 1 above)
Comment (igorkravchenko) [reply GliaWidgets/Sources/RemoteConfiguration/RemoteConfiguration.swift]: Updated 05f24ae04c2e1d1138859cb87a1a4ee052acaf3f.

File: GliaWidgets/Sources/View/Call/Video/FlipCameraButtonStyle.swift [hunk 2]
```diff
@@ -65,23 +65,7 @@ extension FlipCameraButtonStyle.State {
...
-            imageColor: .fill(color: .init(hex: "#FFFFFF"))
-        ),
-        selectedState: State(
-            backgroundColor: .fill(color: .init(hex: "#000000")),
-            imageColor: .fill(color: .init(hex: "#FFFFFF"))
-        ),
-        accessibility: .nop
-    )
+    mutating func apply(_ configuration: RemoteConfiguration.BarButtonStyle?) {
+        activeState.apply(config: configuration)
```
Comment (EgorovEI) [inline GliaWidgets/Sources/View/Call/Video/FlipCameraButtonStyle.swift:69]: If there will be no additional states is future, you can remove `State` and use colors directly.
I mean:
```swift
public struct FlipCameraButtonStyle: Equatable {
         /// Button background color.
        public var backgroundColor: ColorType
        /// Button image color.
        public var imageColor: ColorType
        /// Accessibility related properties.
        public var accessibility: Accessibility
        
        mutating func apply(_ configuration: RemoteConfiguration.BarButtonStyle?) {
             config?.background?.value
                .map { UIColor(hex: $0) }
                .first
                .unwrap { backgroundColor = .fill(color: $0) }

            config?.imageColor?.value
                .map { UIColor(hex: $0) }
                .first
                .unwrap { imageColor = .fill(color: $0) }
        }
}
```
Does it make sense?

Comment (igorkravchenko) [reply GliaWidgets/Sources/View/Call/Video/FlipCameraButtonStyle.swift]: Totally makes sense. Addressed 47071b2c8c333493a2970d12ab9e285886ec2bcf.

**Review by EgorovEI — APPROVED** [review]

**Comment by igorkravchenko** [comment]
!squash

**Pull request merged**